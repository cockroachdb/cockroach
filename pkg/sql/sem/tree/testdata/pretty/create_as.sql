CREATE TABLE t (id) AS VALUES (1);
