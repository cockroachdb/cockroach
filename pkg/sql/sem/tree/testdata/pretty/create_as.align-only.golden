// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
CREATE TABLE t (
	id
) AS
	VALUES
		(
			1
		)

11:
-----------
CREATE TABLE t (
	id
) AS
	VALUES
		(1)

12:
------------
CREATE TABLE t (
	id
) AS
	VALUES (
			1
	       )

14:
--------------
CREATE TABLE t (
	id
) AS
	VALUES (1)

16:
----------------
CREATE TABLE t (
	id
) AS VALUES (1)

29:
-----------------------------
CREATE TABLE t (id) AS VALUES
						(1)

31:
-------------------------------
CREATE TABLE t (id) AS VALUES (
								1
                              )

33:
---------------------------------
CREATE TABLE t (id) AS VALUES (1)


